partner_default_delivery_carrier/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2021 Camptocamp
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Partner Default Delivery Method",
+    "summary": "Allows defining default delivery methods for partners",
+    "version": "16.0.1.0.0",
```

Comment:
This change should go on the migration commit.

Reply:
Done!